platform/mbed_power_mgmt.h

```diff
@@ -68,18 +68,26 @@ extern "C" {
 void sleep_tracker_lock(const char *const filename, int line);
 void sleep_tracker_unlock(const char *const filename, int line);
 
-#define sleep_manager_lock_deep_sleep()           \
-    do                                            \
-    {                                             \
-        sleep_manager_lock_deep_sleep_internal(); \
-        sleep_tracker_lock(__FILE__, __LINE__);   \
-    } while (0);
+#if defined(__CC_ARM)
+#define __FILENAME__ __MODULE__
```

Comment:
Why is this starting with `__` ? This resembles C lib macro but it is not ? As this is in platform header, I expect this goes to the user space (not internal use only).

If it is our own macro, should not use reserved characters (prefix `___`). Similar to what we define in mbed  toolchain (shouldn't this be there rather?)